app/features-json/build_json_controller.rb

```diff
@@ -1,14 +1,16 @@
 require_relative "api_controller"
+require_relative "../features/build_runner/remote_runner"
 require_relative "./view_models/build_summary_view_model"
 require_relative "./view_models/build_view_model"
 
+require "faye/websocket"
+Faye::WebSocket.load_adapter("thin")
+
 module FastlaneCI
   # Controller for providing all data relating to builds
   class BuildJSONController < APIController
     HOME = "/data/projects/:project_id/build"
 
-    use(FastlaneCI::BuildWebsocketBackend)
```

Comment:
We add a controller method in line 114 for the websocket connections instead of using this middleware. The implementation was significantly different, so it made sense to re-implement.